normandy/control/static/control/admin/sass/control.scss

```diff
@@ -1,99 +1,182 @@
+    height: 100%;
+    line-height: 40px;
+    min-width: 165px;
+    position: absolute;
+    right: 0px;
+    top: 0px;
   }
 }
 
-div.breadcrumbs {
-  background: $orange;
-  border-bottom: 1px solid #fff;
-  color: $darkOrange;
-  font-size: 13px;
-  font-weight: 600;
-  line-height: 30px;
-  margin-bottom: 35px;
-  padding: 0px 40px;
-
-  a:link, a:visited {
-    color: $darkOrange;
-    font-weight: 400;
-    opacity: 0.75;
-  }
 
-  a:hover {
-    opacity: 1;
+#secondary-header {
+  background-color: $secondaryColor;
+  border-top: 1px solid #fff;
+  padding: 0px;
+
+  h3 {
+    color: $darkBrown;
+    display: inline-block;
+    float: left;
+    font-size: 14px;
+    font-weight: 600;
+    margin: 15px;
+    padding: 0px 15px;
   }
 
-  .separator {
+  h4 {
     display: inline-block;
-    font-weight: 300;
-    padding: 0px 5px;
+    line-height: 35px;
+    margin: 0 0 0 35px;
   }
 
+  input, .switch {
+    background: rgba(0, 0, 0, 0.1);
+    border: 0px;
+    border-radius: 3px;
+    box-shadow: inset 0 1px 3px rgba(0, 0, 0, 0.3), 0 1px rgba(255, 255, 255, 0.1);
+    font: normal normal 300 13px/22px $OpenSans;
+    min-width: 210px;
+    padding: 5px;
+    position: relative;
+
+    &[type='text'] { padding: 7px 14px; }
+  }
+
+  #filters-container {
+    text-align: right;
+
+    > div {
+      display: inline-block;
+      float: right;
+      margin: 0 0 0 16px;
+    }
+  }
 }
 
-h2 {
-  border-bottom: 1px solid $orange;
-  color: $darkBrown;
-  font-size: 20px;
-  font-weight: 300;
-  margin: 0px;
-  padding: 8px 0px;
 
-  span.bold {
+/* Add & Edit Forms */
+.crud-form {
+  padding: 30px 45px;
+
+  input[type="checkbox"] { opacity: 0; }
+  input[type="number"] { width: 20%; }
+
+  .fluid-1 { width: 6.25%; }
+  .fluid-3 { width: 46.875%; }
+
+  .help {
+    font-size: 11px;
+    margin: 5px 0px;
+    opacity: 0.5;
+  }
+
+  /* Custom checkbox toggle for enabled/disabled state */
+  .checkbox-container {
+    border: 1px solid;
+    border-radius: 3px;
+    float: right;
+    font-size: 12px;
+    line-height: 30px;
+    margin-top: 10px;
+    min-width: 110px;
+    padding: 5px 15px;
+    text-transform: uppercase;
+
+    &:hover { cursor: pointer; }
+  }
+
+  /* Select arrows for toggling locales & countries */
+  .select-arrows {
+    line-height: 35px;
+    margin-top: 30px;
+    text-align: center;
+    width: 6.25%;
+
+    i {
+      display: block;
+      padding: 10px 0px;
+    }
+  }
+
+  a.batch-select {
+    display: block;
+    font-size: 11px;
     font-weight: 600;
+    padding: 8px 0px;
+    text-align: center;
+    text-transform: uppercase;
   }
-}
 
-#content {
-  margin: 40px;
-  min-height: 100%;
-  padding: 0px;
+  /* Action buttons (submit, delete, etc) */
+  .form-action-buttons {
+    border-top: 1px solid $darkCream;
+    line-height: 30px;
+    margin-top: 30px;
+    padding-top: 15px;
 
-  ul.table-list {
-    margin: 0px;
-    padding: 0px;
+    a.delete, input[type="submit"].delete {
```

Comment:
I adjusted the colors a bit to be less muddy and added a super subtle text-shadow. They work better with white text now.

<img width="828" alt="screen shot 2016-04-21 at 11 53 53 am" src="https://cloud.githubusercontent.com/assets/856935/14715717/d51aa4e2-07b7-11e6-8832-403a10c7af2e.png">